zerver/management/commands/restore_messages.py

```diff
@@ -50,5 +56,7 @@ def handle(self, **options: Any) -> None:
             restore_data_from_archive_by_realm(realm)
         elif options["transaction_id"]:
             restore_data_from_archive(ArchiveTransaction.objects.get(id=options["transaction_id"]))
-        else:
+        elif options["restore_all"]:
             restore_all_data_from_archive(restore_manual_transactions=options["restore_deleted"])
+        else:
+            raise CommandError("An argument needs to be specified.")
```

Comment:
Maybe be more specific?
```suggestion
            raise CommandError("You must specify one of --transaction-id, --realm, or --restore-all.")
```